flycheck.el

```diff
@@ -4257,6 +4263,27 @@ universal prefix arg, and only the id with normal prefix arg."
     (`(eval ,_) t)
     (_ nil)))
 
+(defun flycheck-default-directory-wrapper (checker)
```

Comment:
I don't like the name of this function :)  It's too generic.

Can we come up with a better one?  E.g. `flycheck-compute-default-directory`, or so?